solr/solrj/src/test/org/apache/solr/client/solrj/impl/CloudHttp2SolrClientTest.java

```diff
@@ -246,6 +257,52 @@ public void testAliasHandling() throws Exception {
         2, client.query(null, paramsWithMixedCollectionAndAlias).getResults().getNumFound());
   }
 
+  @Test
+  @LogLevel("org.apache.solr.servlet.HttpSolrCall=DEBUG")
+  public void testHttpCspPerf() throws Exception {
+
+    String collectionName = "HTTPCSPTEST";
+    CollectionAdminRequest.createCollection(collectionName, "conf", 2, 1).process(cluster.getSolrClient());
+    cluster.waitForActiveCollection(collectionName, 2, 2);
+
+    CloudSolrClient solrClient = null;
+    try (LogListener entireClusterStateLogs = LogListener.info(HttpSolrCall.class).regex(PATTERN_WITHOUT_COLLECTION); LogListener collectionClusterStateLogs = LogListener.info(HttpSolrCall.class).regex(PATTERN_WITH_COLLECTION); LogListener adminRequestLogs = LogListener.info(HttpSolrCall.class).substring("[admin]")) {
```

Comment:
try-with-resources opportunity here for this CloudSolrClient

Reply:
Updated the PR